wazo_call_logd/plugins/cdr/api.yml

```diff
@@ -35,6 +35,33 @@ paths:
       produces:
         - application/json
         - text/csv; charset=utf-8
+  /cdr/statistics:
+    get:
+      summary: Statistics based on CDR
+      description: '**Required ACL:** `call-logd.cdr.read`'
+      tags:
+        - cdr
+      parameters:
+      - $ref: '#/parameters/tenantuuid'
+      - $ref: '#/parameters/from'
+      - $ref: '#/parameters/until'
+      - $ref: '#/parameters/interval'
+      - $ref: '#/parameters/call_direction'
+      - $ref: '#/parameters/number'
+      - $ref: '#/parameters/tags'
+      - $ref: '#/parameters/user_uuid'
+      - $ref: '#/parameters/recurse'
```

Comment:
Do we really want to support recurse for this view?

Reply:
Removed, right now I don't see the need for a `recurse`